README.md

```diff
@@ -1,65 +1,86 @@
-# React and Redux
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-If you are not familiar with linters and GitHub Actions, read [root level README](../README.md).
+# Metrics webapp - TV Shows
 
-## Set-up GitHub Actions
+> "METRIC WEBAPP" the Metric WebApp is an app that displays TV shows gotten from an TVMAZE API. The API request contains information about a particular Show such as movie description, ratings, duration and release date. The homepage of the App groups the movie into categories based on on Show genres. it also give a metric figure(numbers) of movies available in each category
 
-This GitHub Action is going to run [ESLint](https://eslint.org/) and [Stylelint](https://stylelint.io/) to help you find style issues.
+- Display Show categoriess.
+- Gives details of shows.
+- search shows.
 
-[Stylelint](https://stylelint.io/) is a linter for your stylesheets that helps you avoid errors and enforce conventions.
+## Built With
 
-[ESLint](https://eslint.org/) is a tool for identifying and reporting on patterns found in ECMAScript/JavaScript code, with the goal of making code more consistent and avoiding bugs.
+- React
+- Jest-test
+- Redux
 
-Please do the following **steps in this order**:
+## Live Demo
 
-1. In the first commit of your feature branch create a `.github/workflows` folder and add a copy of [`.github/workflows/linters.yml`](.github/workflows/linters.yml) to that folder.
-    - **Remember** to use the file linked above
-    - **Remember** that `.github` folder starts with a dot.
-2. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
-    - If you think that change is necessary - open a [Pull Request in this repository](../README.md#contributing) and let your code reviewer know about it.
-3. When you open your first pull request you should see the result of the GitHub Actions:
+## Video [Link](https://drive.google.com/file/d/1geftNJtwXUKpLTZl-mva0roMZIWgkD53/view?usp=sharing)
 
-![gh actions checks](../assets/images/gh-actions-eslint-stylelint-checks.png)
+[Netlify Live Demo Link]()
```

Comment:
- [ ] Your live link is not working, this is due to the fact that your netlify is failing kindly run npm run build in your terminal to know where the error is coming from before your next deploy